File: MODUL 4 PEMROGRAMAN 1/PRAK402-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
int main (){
    int i, inp, ganjil, genap;
    for(i = 1; i <= 3; i++){
        scanf("%d", &inp);

        for (ganjil = 1; ganjil <= inp; ganjil++){
            if(ganjil % 2 != 0){
                printf("%d ",ganjil);
            }
        }
        printf("\n");
        for(genap = inp; genap >= 2; genap--){
            if(genap % 2 == 0){
                printf("%d ", genap);
            }
        }  printf("\n");
    } return 0;
}

File: MODUL 4 PEMROGRAMAN 1/PRAK402-2310817120002-SITI RATNA DWINTA SARI.py
for i in range(3):
    inp = int(input())

    for ganjil in range(1, inp + 1):
        if ganjil % 2 != 0:
            print(ganjil, end=" ")

    print()  
    for genap in range(inp, 1, -1):
        if genap % 2 == 0:
            print(genap, end=" ")

    print()  
